src/server/apis/settings.ts

```diff
@@ -13,13 +13,18 @@ function getRouter(github: GitHub): express.Router {
   settingsRouter.post(
       '/orgs.json',
       async (request: express.Request, response: express.Response) => {
-        const loginDetails = await userModel.getLoginFromRequest(request);
+        const loginDetails =
+            await userModel.getLoginFromRequest(github, request);
         if (!loginDetails) {
           response.sendStatus(400);
           return;
         }
 
         const scopes = loginDetails.scopes;
+        if (!scopes) {
```

Comment:
This doesn't return a response. If `null` is removed, this line won't be necessary, and it'll fall into the below 400 statement of missing scope.